.github/workflows/ci.yml

```diff
@@ -27,16 +27,17 @@ jobs:
 
     steps:
       - name: Clone repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python }}
-          cache: pip
 
-      - run: python --version
-      - run: pip --version
+      - name: Print versions
```

Comment:
This is also slow on macOS and Windows. We can already see the Python version, so maybe we should drop this step?